client/src/components/Header/Header.js

```diff
@@ -21,7 +22,12 @@ const Header = () => {
     <div id="header" className="header">
       <div className="header-content">
         <Link to="/">
-          <div className="header-font">WATER THE TREES</div>
+          <img
+            className="header--logo"
+            src={WttHalloween}
+            alt="WTT Halloween"
+          />
+          {/* <div className="header-font">Water The Trees</div> */}
```

Comment:
Can you please remove the commented out code? 

@ri0nardo - I think it would be awesome to have a default, non-themed logo + favicon so we only need to replace the images each time.

Reply:
Okay @jazhen , will do that now